plugins/ui-kit-candidates/SortableColumnHeaderCellIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+
+type SortDirection = "ASC" | "DESC";
```

Comment:
Don't we have this already in `plugins/nodes/src/js/types/SortDirection.tsx`

Reply:
sure, but this is going to move into ui kit, from there we cant link into `dcos-ui/plugins/nodes`

Reply:
ok 👍